### DISCUSSION TITLE
Configuration in a .Net Core Console application

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#12] [BlogPosts]
# Configuration in a .Net Core Console application

Adding configuration to a .Net core Console application to read configuration from a json file, or environmental variables or command line arguments.

https://pradeepl.com/blog/dotnet/configuration-in-a-net-core-console-application/

**Comment by alimajedha-home** [comment]
Hello, how to use Environment variables in connection string?
